api/appsettings.go

```diff
@@ -16,11 +16,17 @@ type AppSettings struct {
 	// Maintenance determines if the application is taken down for maintenance or not.
 	Maintenance *bool `json:"maintenance,omitempty"`
 	// Routable determines if the application should be exposed by the router.
-	Routable *bool `json:"routable,omitempty"`
+	Routable  *bool    `json:"routable,omitempty"`
+	Whitelist []string `json:"whitelist,omitempty"`
```

Comment:
Can you add a commend that whitelist is read only here and to look at the whitelist package instead?